README.md

```diff
@@ -1 +1,41 @@
 # TZDatePicker
+
+is a simple DatePicker that easy to use
```

Comment:
This is not a good sentence, and also not the reason I created this pod. I created it because I wanted a date picker with an optional year. Please change this into "This is a replacement for the UIDatePicker, which allows the year to be optional."